pkg/scheduler/core/generic_scheduler.go

```diff
@@ -593,11 +593,18 @@ func addNominatedPods(pod *v1.Pod, meta predicates.PredicateMetadata,
 	podsAdded := false
 	for _, p := range nominatedPods {
 		if util.GetPodPriority(p) >= util.GetPodPriority(pod) && p.UID != pod.UID {
+			outCopy := nodeInfoOut.Clone()
```

Comment:
The previous version was good. I rethink it a little bit, we don't need to ensure the two operations as a transaction.

Can you revert to the previous version? (just simply log the error)